### DISCUSSION TITLE
Angular [enabled] property

### DISCUSSION THREAD

**Discussion by zygimantas** [nolimits4web/swiper#4756] [Q&A]
Is there a reason why [enabled] attribute is missing in Angular library?

In documentation, it is written:

> SwiperComponent props
> Swiper Angular component receive all Swiper parameters as component props

But not "enabled".

**Comment by nolimits4web** [comment]
cc @vltansky

**Reply by vltansky** [reply]
Issue: https://github.com/nolimits4web/swiper/issues/4948
PR: https://github.com/nolimits4web/swiper/pull/4949
